lib/reporting/drop_off_report.rb

```diff
@@ -0,0 +1,397 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+  require 'reporting/identity_verification_report'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
+
+module Reporting
+  class DropOffReport < IdentityVerificationReport
```

Comment:
yeah i was trying to leverage some of the similar pieces like the `cloudwatch` method (esp since i first thought they were more similar) but i can just pull the necessary code out

Reply:
updated https://github.com/18F/identity-idp/pull/9989/commits/6aaa8ec6ddfd8ec047ea564c89aa158836607f65